pkg/serving/config_changes.go

```diff
@@ -27,6 +27,15 @@ import (
 	servingv1beta1 "knative.dev/serving/pkg/apis/serving/v1beta1"
 )
 
+// VolumeSourceType is a type standing for enumeration of ConfigMap and Secret
+type VolumeSourceType int
+
+// Enumeration list of Kay-Value sources: ConfigMap or Secret
```

Comment:
Key-Value?

Reply:
Yes. Key-Value. But, I've changed the entire comments to clarify it.